fvm/scriptEnv.go

```diff
@@ -660,70 +660,43 @@ func (e *ScriptEnv) ImplementationDebugLog(message string) error {
 	return nil
 }
 
-func (e *ScriptEnv) ProgramParsed(location common.Location, duration time.Duration) {
+func (e *ScriptEnv) RecordTrace(operation string, location common.Location, duration time.Duration, logs []opentracing.LogRecord) {
 	if e.isTraceable() {
```

Comment:
Not really related to the new changes; maybe inverting the condition would be a good idea to reduce nested levels?

Reply:
Smart :D

Reply:
updated